src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,6 +50,23 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
+    /**
+     * Function removes the comma in the address argument
+     * Initializes classes: Block, Street, Unit, PostalCode
+     */
```

Comment:
The start of this header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)